demo/templates.jl

```diff
@@ -0,0 +1,70 @@
+include("parser.jl")
+
+@widget FilePicker """<input type='file' accept='{{#accepts}}{{.}}, {{/accepts}}'>""" accepts=[] default=Dict("name" => "", "data" => UInt8[], "type" => "")
```

Comment:
I'm pretty sure you would want `mt"""..."""` in these definitions so the parsing comes once at compile time and not each time at run time. Should be faster, though perhaps not noticeable.

Reply:
Perfect, I do not even need to call the parser explicitly. Thanks!